httpSymphonyClient.go

```diff
@@ -252,11 +252,10 @@ func (client *Client) startTicker() {
 // the client worker goroutine
 // does the sending of data to the server
 func (client *Client) clientWorker() {
-	closeHttp := func(r *http.Response) {
+	closeit := func(r *http.Response, buf *logBuffer) {
 		r.Body.Close()
-	}
-	closeBuf := func(buf *logBuffer) {
 		greasego.RetireCallbackData(buf.data)
+		debugging.DEBUG_OUT("  OKOKOKOKOKOKOKOKOK -----> retired callback data\n\n")
```

Comment:
Can we clean up these DEBUG_OUT msgs instead of saying OKOK....

Reply:
I am not cleaning any of the debug messages added by Ed, will pick that cleanup once the functionality is done and bugs are fixes